### PR TITLE
[C-1671] Finalize wallet-connect flow

### PR DESCRIPTION
**Pull request by dylanjeffers**
### Description

Finalizes wallet connect flow ui, logic, edge-cases

Android still needs a once-over

### CODE REVIEW

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/dj-c-1671-finalize-ios-wallet-connect

File: packages/mobile/src/screens/wallet-connect/components/LinkedWallet.tsx [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+    borderRadius: 14,
+    padding: 2,
+    marginRight: spacing(2)
+  },
+  address: {
+    flexDirection: 'row',
+    alignItems: 'center'
+  },
+  copyIcon: {
+    marginBottom: 2,
```
Comment (Kyle-Shanks) [inline packages/mobile/src/screens/wallet-connect/components/LinkedWallet.tsx:70]: spacing

File: packages/mobile/src/screens/wallet-connect/types.ts [hunk 2]
```diff
@@ -14,7 +19,11 @@ export type SignedMessageParams = {
 }
 
 export type WalletConnectParamList = {
-  Wallets: undefined | ConnectNewWalletParams | SignedMessageParams
+  Wallets:
```
Comment (Kyle-Shanks) [inline packages/mobile/src/screens/wallet-connect/types.ts:22]: is there a reason for undefined vs `?:` here?

Comment (dylanjeffers) [reply packages/mobile/src/screens/wallet-connect/types.ts:22]: yeah good q, i tihnk the undefined can be removed actually, i think the idea here is that params can be undefined, but they are always present? though in reality that means the same thing! will look into

**Review by Kyle-Shanks — APPROVED** [review]
Looks good. awesome stuff

**Pull request merged**